lib/ansible/modules/network/netvisor/_pn_cluster.py

```diff
@@ -29,6 +29,10 @@
 author: "Pluribus Networks (@amitsi)"
 version_added: "2.2"
 short_description: CLI command to create/delete a cluster.
+deprecated:
+  removed_in: '2.12'
+  why: Doesn't support latest Pluribus Networks netvisor
+  alternative: Latest modules will be pushed in ansible 2.9 instead.
```

Comment:
```suggestion
  alternative: Latest modules will be pushed in Ansible 2.9 instead.
```

Reply:
@gundalow @Akasurde  i have made the required changes. :)

Reply:
Can you please let me know what needs to be added in `porting_guide_2.8.rst`?

Reply:
You need to add a note here - https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.8.rst#deprecation-notices

Reply:
@rajaspachipulusu17 you need to rebase the branch - see http://docs.testing.ansible.com/ansible/latest/dev_guide/developing_rebasing.html